sdk/src/ctx.rs

```diff
@@ -69,3 +70,27 @@ pub trait TangleClientContext {
         &self,
     ) -> impl Future<Output = Result<subxt::OnlineClient<Self::Config>, subxt::Error>>;
 }
+
+/// `ServicesContext` trait provides access to the current service and current blueprint from the context.
+pub trait ServicesContext {
```

Comment:
@shekohex we also need the restake distribution of the operators of the service instance. This should contain the assets chosen on the service too.